scripts/ghidra/PatchestryDecompileFunctions.java

```diff
@@ -156,6 +203,167 @@ private HighVariable lValueOf(Varnode node) throws Exception {
         	return var;
         }
 
+        private String getDisplayName(DataType data_type) throws Exception {
```

Comment:
I don't think we need this? If we go straight to a `TypedefDecl` in Clang ASTs, for example, then it won't stop us from putting spaces in the name, even it it is bizarre seeming. It seems plausible to me that someone might choose to name a typedef like `pair<int, int>` as a kind of "pun" for templates, and we should allow that without removing the space.